zgw/zaken-api/src/test/kotlin/com/ritense/zakenapi/service/ZaakTypeLinkServiceTest.kt

```diff
@@ -111,7 +111,7 @@ class ZaakTypeLinkServiceTest {
 
     @Test
     fun `should get empty zaakTypeLinks`() {
-        whenever(processDocumentAssociationService.findProcessDocumentDefinitions("documentDefinitionName", true))
+        whenever(processDocumentAssociationService.findProcessDocumentDefinitions("documentDefinitionName", null, null))
```

Comment:
The fact that this is needed means the method is breaking, right? Maybe we should overload this?

Reply:
yes, maybe that is true. I will have a look

Reply:
@ivo-ritense suggestions are welcome :-)

Reply:
@ivo-ritense I removed the mock as it does not seem to be needed/used by this specific unit test